<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="titleEdit">
        <property name="placeholderText">
         <string>Title</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="typeCombo">
        <item>
         <property name="text">
          <string>全部</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>唐诗</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>宋词</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>宋诗</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>五代·花间集</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>五代·南唐二主词</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>论语</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>诗经</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>大学</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>孟子</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>中庸</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>三字经</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>千字文</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="authorEdit">
        <property name="placeholderText">
         <string>author</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="contentEdit">
        <property name="placeholderText">
         <string>content</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="queryButton">
        <property name="text">
         <string>Query</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="viewButton">
        <property name="text">
         <string>View</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTableView" name="tableView">
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectItems</enum>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>true</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="perPageButton">
        <property name="text">
         <string>Pervious Page</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="pageNumSpin">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="suffix">
         <string/>
        </property>
        <property name="prefix">
         <string>Page </string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="jumpToPageButton">
        <property name="text">
         <string>Jump To:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nextPageButton">
        <property name="text">
         <string>Next Page</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Poem number of per page:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="poemNumPerPageSpin">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="comfireButton">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>titleEdit</tabstop>
  <tabstop>typeCombo</tabstop>
  <tabstop>authorEdit</tabstop>
  <tabstop>contentEdit</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>queryButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>onQueryButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>593</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>perPageButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>onPerPageButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>593</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextPageButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>onNextPageButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>593</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>onViewButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>593</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>jumpToPageButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>onJumpToButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>712</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comfireButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>onComfireButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>645</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onQueryButtonClicked()</slot>
  <slot>onViewButtonClicked()</slot>
  <slot>onPerPageButtonClicked()</slot>
  <slot>onNextPageButtonClicked()</slot>
  <slot>onJumpToButtonClicked()</slot>
  <slot>onComfireButtonClicked()</slot>
 </slots>
</ui>
